index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Parser } from 'acorn' 
+
+declare const jsx: (options?: jsx.AcornJSXOptions) => (BaseParser: typeof Parser) => typeof Parser;
```

Comment:
Note there is 2nd parameter in the function too.

Reply:
I can’t tell this from the code or the readme.

https://github.com/acornjs/acorn-jsx/blob/52a64313bb95aba5fbfddd1b0256aa0fce229bb2/index.js#L73-L81

Reply:
Oh, sorry, I looked up the wrong function :) https://github.com/acornjs/acorn-jsx/blob/52a64313bb95aba5fbfddd1b0256aa0fce229bb2/index.js#L93